pkg/api-server/testdata/inspect_retry.json

```diff
@@ -0,0 +1,30 @@
+[
```

Comment:
As we are returning an array, maybe it would be better to follow the same pattern as we have when returning other arrays like

```
total: Number
items: Array<Item>
``` 
WDYT?